pontoon/pretranslation/transformer.py

```diff
@@ -135,6 +135,16 @@ def __init__(
         self.locale = locale
         self.services: list[str] = []
 
+    def visit_Attribute(self, node):
+        is_accesskey = (
+            node.id.name.endswith("accesskey")
+            and isinstance(node.value.elements[0], FTL.TextElement)
+            and len(node.value.elements[0].value) <= 1
```

Comment:
This condition is either insufficient or too strict; we've plenty of accesskeys that are selectors on `PLATFORM()`. For example: https://pontoon.mozilla.org/en-GB/firefox/browser/browser/menubar.ftl/?string=220792

Given that this is gated on `accesskey_localization` which defaults to `True`, I don't think we need to look at the `.accesskey` value if the flag is set to `False`.